components/layout/fragment.rs

```diff
@@ -186,6 +186,8 @@ pub enum SpecificFragmentInfo {
     Multicol,
     MulticolColumn,
     UnscannedText(Box<UnscannedTextFragmentInfo>),
+
```

Comment:
It could be worth either writing down here how these are used, or maybe renaming it to `TruncatedTextFragment`?